decidim-core/lib/decidim/events/simple_event.rb

```diff
@@ -116,6 +103,18 @@ def resource_text
         nil
       end
 
+      def resource_title
```

Comment:
There were some loading issues when i have backported the PR, and i had to add the overrides within this file. This is why there are some changes here, that re not present in the initial PR. It may be related to https://github.com/decidim/decidim/pull/12157 but i am 100% sure.

Reply:
Ok, as I mentioned in another comment, if there's a good reason let's leave it as is